config/routes.rb

```diff
@@ -479,8 +479,8 @@
 
   # Licences
   %w[licence license].each do |spelling|
-    get "#{spelling}s/cc-sa-4", to: redirect("docs/using/licenses/cc-by-nc-sa-4"), as: "cc_sa_4_#{spelling}"
-    get "#{spelling}s/cc-by-nc-sa-4", to: redirect("docs/using/licenses/cc-by-nc-sa-4"), as: "cc_by_nc_sa_4_#{spelling}"
```

Comment:
The docs file does not have the `-4` suffix. Neither does the `AGPL 3` link (it's just `agpl`), so I fixed it here instead of changing the filename in the docs.